integration/test-cleanup.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+apps=$( auth0 apps list --format json --no-input)
+
+for app in $( echo "${apps}" | jq -r '.[] | @base64' ); do
+    _jq() {
+     echo "${app}" | base64 --decode | jq -r "${1}"
+    }
+
+    clientid=$(_jq '.ClientID')
+    name=$(_jq '.Name')
+
+    if [[ $name != "Default App" ]]
```

Comment:
TODO: I'm going to create the test apps with a common prefix and update this logic so only integration test apps will be deleted.